From c18a55f9ef9969b38e5cf7f0f319b8ec5df34809 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 9 Jul 2018 12:07:30 +0000
Subject: [PATCH] Unbreak with boost 1.68

In file included from src/main.cc:34:
In file included from src/global.h:41:
In file included from src/option.h:45:
In file included from src/scope.h:45:
In file included from src/op.h:45:
In file included from src/expr.h:45:
In file included from src/exprbase.h:57:
src/utils.h:47:10: fatal error: 'boost/uuid/sha1.hpp' file not found
 #include <boost/uuid/sha1.hpp>
          ^
---
 src/utils.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/utils.h b/src/utils.h
index b21dff7af..857b8289c 100644
--- a/src/utils.h
+++ b/src/utils.h
@@ -44,7 +44,11 @@
 #ifndef _UTILS_H
 #define _UTILS_H
 
+#if (BOOST_VERSION >= 106600)
+#include <boost/uuid/detail/sha1.hpp>
+#else
 #include <boost/uuid/sha1.hpp>
+#endif
 
 /**
  * @name Default values
